IDontHaveVinigrate5_1.py

```diff
@@ -0,0 +1,17 @@
+from datetime import date
+import datetime
+import random
+
+date_entry = input('Enter a date in YYYY-MM-DD format ')
```

Comment:
Hi Reut :)
Nice job, but we should do it slightly different - 
Please write your code in a function (or divide it into several functions if needed).
The functions should be called in main().
Also, please remember to add documantation to the functions you are about to write.